### PR TITLE
Test metrics from Observability guides are available

### LINKED ISSUE #35807: Add a test that the metrics listed in the docs are available from Keycloak (keep it simple, ignore metrics that don't show up right after the start)

### PR DESCRIPTION
**Pull request by mhajas**
Closes #35807

### CODE REVIEW

**Review by mabartos — DISMISSED** [review]

File: quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/MetricsDistTest.java [hunk 1]
```diff
@@ -60,9 +60,47 @@ void testMetricsEndpointNotEnabled(KeycloakDistribution distribution) {
     void testMetricsEndpoint() {
         when().get("/metrics").then()
                 .statusCode(200)
-                .body(containsString("jvm_gc_"))
-                .body(containsString("http_server_active_requests"))
-                .body(containsString("vendor_statistics_hit_ratio"))
+
+                // Test metrics used in Observability guides are present
+                // SLIs
+                .body(containsString("TYPE http_server_requests_seconds summary"))
```
Comment (ahus1) [inline quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/MetricsDistTest.java:66]: I see that this test fails. AFAIK this metric only appears on the second request, that's why I added a dummy request below in the next text case in this same file.

**Review by ahus1 — CHANGES_REQUESTED** [review]
Thank you for the PR. See below for a comment about the failing test case.

File: quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/MetricsDistTest.java (hunk 1 above)
Comment (mhajas) [reply quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/MetricsDistTest.java:70]: Sorry I missed your comment and fixed by sending another metric request before assertion. Let me know whether it works for you or not.

Comment (ahus1) [reply quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/MetricsDistTest.java:70]: LGTM, this is what I had in mind. 

PS: I tend to comment on PRs that are in "ready to review state" :-)

**Review by ahus1 — APPROVED** [review]

File: quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/MetricsDistTest.java (hunk 1 above)
Comment (mhajas) [reply quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/MetricsDistTest.java:70]: > PS: I tend to comment on PRs that are in "ready to review state" :-)

I did not expect this PR to fail so I did not send it as a draft, then I would do the green light for reviews by switching from draft to ready for review. I tried 100 repetitions locally. It is interesting the test was failing only when running in the container.

**Review by ahus1 — APPROVED** [review]

**Pull request merged**